pkg/controller/backup_controller.go

```diff
@@ -641,40 +641,40 @@ func persistBackup(backup *pkgbackup.Request,
 	csiVolumeSnapshots []*snapshotv1beta1api.VolumeSnapshot,
 	csiVolumeSnapshotContents []*snapshotv1beta1api.VolumeSnapshotContent,
 ) []error {
-	errs := []error{}
+	persistErrs := []error{}
 	backupJSON := new(bytes.Buffer)
 
-	if err := encode.EncodeTo(backup.Backup, "json", backupJSON); err != nil {
-		errs = append(errs, errors.Wrap(err, "error encoding backup"))
+	if errs := encode.EncodeTo(backup.Backup, "json", backupJSON); errs != nil {
```

Comment:
this one should just be `err`, since it's only a single error

Reply:
I knew selecting the region and doing replace-all was going to bite me somehow.

Reply:
😂